### DISCUSSION TITLE
Feature Request: Eraser

### DISCUSSION THREAD

**Discussion by MartinStuwe** [zsviczian/obsidian-excalidraw-plugin#864] [Ideas]
I want to add a new option to the eraser, where it is basically like a white pencil. So that one can select between the auto eraser and the eraser without auto selection.

In which file can I find the eraser methods and the menu above where the eraser icon is placed?

**Comment by wwf194** [comment]
This would be very useful, for example, when drawing a dashed curve. I really hope this could be implemeted.
